statements/queries.go

```diff
@@ -9,6 +9,13 @@ SELECT
 	datname
 FROM
 	pg_database`
+
+	// DatabaseCount clause
+	DatabasesCount = `
```

Comment:
Can be reused to query, so avoid getting duplicating `DatabasesSelect` and `SchemasSelect`
Creating a variable for the fields